app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,16 @@
+class PostsController < ApplicationController
+  def new
+  end
+  def show
+    @post = Post.find(params[:id])
+  end
+  def create
+    @post = Post.new(post_params)
+    @post.save
+    redirect_to @post
+  end
+  private
```

Comment:
There should be a space between the two methods. Please follow rubocop standards.